lib/modules/manager/github-actions/extract.ts

```diff
@@ -98,11 +101,11 @@ function extractWithRegex(content: string): PackageDependency[] {
       } else if (shaShortRe.test(currentValue)) {
         dep.currentValue = tag;
         dep.currentDigestShort = currentValue;
+      } else if (customVersioningRe.test(currentValue)) {
+        dep.currentValue = currentValue;
       } else {
         dep.currentValue = currentValue;
-        if (!dockerVersioning.api.isValid(currentValue)) {
-          dep.skipReason = 'invalid-version';
-        }
+        dep.skipReason = 'invalid-version';
```

Comment:
Why this change?

Reply:
This "docker...isValid" part is not relevant anymore because `customVersioningRe` will do the same 
```
} else if (customVersioningRe.test(currentValue)) {
        dep.currentValue = currentValue;
``` 
I should rename this variable.

Reply:
I think we should accept any value and leave it to later versioning to decide if it's a valid version.

that way we can support almost all version schemes.

Reply:
This PR is more simple now :)
If everything is ok, I will edit the PR description.